src/cli/java/org/commcare/util/screen/EntityScreen.java

```diff
@@ -192,8 +192,13 @@ public Subscreen getCurrentScreen() {
         return mCurrentScreen;
     }
 
+    protected String getReturnValueFromSelection(TreeReference contextRef) {
+        return getReturnValueFromSelection(contextRef, mNeededDatum, evalContext);
```

Comment:
Pre-existing nitpick, but I wish this class were consistent about using the `m` prefix for class attributes.